config/software/server-jre.rb

```diff
@@ -36,12 +38,33 @@
 whitelist_file "jre/plugin"
 whitelist_file "jre/bin/appletviewer"
 
+license_warning = "By including the JRE, you accept the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, which can be found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html"
+
+version "8u91" do
+  # https://www.oracle.com/webfolder/s/digest/8u91checksum.html
+  if _64_bit?
+    source url: "http://download.oracle.com/otn-pub/java/jdk/8u91-b14/server-jre-8u91-linux-x64.tar.gz",
+           md5: "3f3d7d0cd70bfe0feab382ed4b0e45c0",
+           cookie:  "gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie",
```

Comment:
Can we DRY things up here and add a `accept_license_cookie` var or similar (like you did with `license_warning`)?